benchmarks/haskell16/pos/Overview.hs

```diff
@@ -64,8 +68,9 @@ fib_incr :: Int -> Proof
 {-@ fib_incr :: n:Nat -> {fib n <= fib (n+1)} @-}
 fib_incr n
    | n == 0
-   = toProof $
-      fib 0 <! fib 1
+   = fib 0 <! fib 1
+   ! QED
+
```

Comment:
nice! But too many `!` -- can you pick something else e.g. `*** QED` ?

On Wed, May 18, 2016 at 12:52 PM, Niki Vazou <notifications@github.com>
wrote:

> In benchmarks/haskell16/pos/Overview.hs
> <https://github.com/ucsd-progsys/liquidhaskell/pull/715#discussion_r63769996>
> :
>
> > @@ -64,8 +68,9 @@ fib_incr :: Int -> Proof
> >  {-@ fib_incr :: n:Nat -> {fib n <= fib (n+1)} @-}
> >  fib_incr n
> >     | n == 0
> > -   = toProof $
> > -      fib 0 <! fib 1
> > +   = fib 0 <! fib 1
> > +   ! QED
> > +
>
> @ranjitjhala <https://github.com/ranjitjhala> what do you think of this
> QED syntax?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsd-progsys/liquidhaskell/pull/715/files/e96b638d95411b17f4584ddcc436856cd3554957..738a7cd4d581180d1a53c8b8a01ddfae46cc73df#r63769996>
>